Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -174,6 +175,9 @@ const TouchableHighlight = createReactClass({
      * shiftDistanceY: Defaults to 2.0.
      * tiltAngle: Defaults to 0.05.
      * magnification: Defaults to 1.0.
+     * pressMagnification: Defaults to 1.0.
```

Comment:
Ditto. These will need to be copied over to the `react-native-website` repo as well.